src/main/kotlin/net/lumalyte/lg/interaction/menus/guild/GuildWarManagementMenu.kt

```
@@ -200,43 +210,678 @@ class GuildWarManagementMenu(private val menuNavigator: MenuNavigator, private v
+        gui.addPane(pane)
+
+        // Center items
+        val totalItem = ItemStack.of(Material.BOOK)
+            .name(lang.gui("menu.guild_war_management.war_stats.total", "count" to total))
+            .lore(lang.gui("menu.guild_war_management.war_stats.wins", "count" to wins))
+            .lore(lang.gui("menu.guild_war_management.war_stats.losses", "count" to losses))
+            .lore(lang.gui("menu.guild_war_management.war_stats.draws", "count" to draws))
+        pane.addItem(GuiItem(totalItem) { }, 1, 1)
+
+        val ratioItem = ItemStack.of(Material.GOLDEN_APPLE)
+            .name(lang.gui("menu.guild_war_management.war_stats.win_rate", "rate" to String.format("%.1f", winLossRatio * 100)))
```

detekt (1.23.8)

[warning] String.format("%.1f", winLossRatio * 100) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)